pkgs/development/python-modules/linkedin-messaging/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, python3 }:
```

```suggestion
{ lib, fetchFromGitHub }:
```
This is now allowed in pythonPackages because it would always draw in python3(9). Please list the inputs individually.